seatunnel-api/src/main/java/org/apache/seatunnel/api/transform/SeaTunnelTransform.java

```diff
@@ -54,4 +55,8 @@ default SchemaChangeEvent mapSchemaChangeEvent(SchemaChangeEvent schemaChangeEve
 
     /** call it when Transformer completed */
     default void close() {}
+
+    void setMetricsContext(MetricsContext metricsContext);
+
+    MetricsContext getMetricsContext();
```

Comment:
Transform does not seem to need to abstract `MetricsContext` and expose it through this interface.

Reply:
Need to set the MetricsContext in TransformFlowLifeCycle Constructor.getMetricsContext does not need.

Reply:
`set` is mislabel [/cry]